test/integration/TokenAssociateIntegrationTest.js

```diff
@@ -134,6 +141,739 @@ describe("TokenAssociate", function () {
         }
     });
 
+    describe("Max Auto Associations", function () {
+        let receiverKey, receiverId;
```

Comment:
I think these values should be created in every test case. What do you think @svetoslav-nikol0v ?

Reply:
After you noted it, yes, the reason is that updating the account on the previous test may cause a false positive on the next one. @ivaylonikolov7

Reply:
You're right. Ignore this.